### PR TITLE
deprecated functions with the warning statement for computing delay, …

### PR DESCRIPTION
**Pull request by sgratiy**
…pause, burst, has_delay, has_burst, has_pause features as well as plotting them

### CODE REVIEW

File: allensdk/ipfx/ephys_extractor.py [hunk 1]
```diff
@@ -319,6 +319,7 @@ def pause(self, t, spikes_df, start, end, cost_weight=1.0):
     max_reliability : max fraction of times most reliable pause was detected given weights tested
     n_max_rel_pauses : number of pauses detected with `max_reliability`
     """
+    logging.warning("This function will be removed")
```
Comment (dyf) [inline allensdk/ipfx/ephys_extractor.py]: replace `logging.warning` with `warnings.warning`

https://docs.python.org/2/howto/logging.html

> warnings.warn() in library code if the issue is avoidable and the client application should be modified to eliminate the warning

File: allensdk/ipfx/ephys_extractor.py [hunk 2]
```diff
@@ -344,6 +345,7 @@ def burst(t, spikes_df, tol=0.5, pause_cost=1.0):
     max_burstiness_index : max "burstiness" index across detected bursts
     num_bursts : number of bursts detected
     """
+    logging.warning("This function will be removed")
```
Comment (nicain) [inline allensdk/ipfx/ephys_extractor.py]: Here too

File: allensdk/ipfx/ephys_extractor.py [hunk 3]
```diff
@@ -374,6 +376,8 @@ def delay(t, v, spikes_df, start, end):
     delay_ratio : ratio of latency to tau (higher means more delay)
     tau : dominant time constant of rise before spike
     """
+    logging.warning("This function will be removed")
+
```
Comment (nicain) [inline allensdk/ipfx/ephys_extractor.py]: Here too

File: allensdk/ipfx/stimulus_protocol_analysis.py [hunk 4]
```diff
@@ -262,7 +260,6 @@ def analyze(self, sweep_set):
                 common_amp = c[0]
 
         ca_features = spiking_features[spiking_features["stim_amp"] == common_amp]
-        print "common_amp:",common_amp
```
Comment (nicain) [inline allensdk/ipfx/stimulus_protocol_analysis.py]: Should lis be turned into a logging call at debug level?

**Pull request merged**